### PR TITLE
Corrin rework

### PR DESCRIPTION
**Pull request by SkewedAskew**
After some time in the oven, the corn has become buttery and delicious

Changelog: [Corrin_changes.txt](https://github.com/HDR-Development/HewDraw-Remix/files/9401887/Corrin_changes.txt)

These romfs are needed for the pin jump fix: [hdr-assets-pr.zip](https://github.com/HDR-Development/HewDraw-Remix/files/9388366/hdr-assets-pr.zip)

Hitbox Comparison: https://imgur.com/a/urtiFxa

### CODE REVIEW

**Review by WuBoytH — DISMISSED** [review]

**Comment by yoonikoo** [comment]
There are some things that need to be fixed. Just discuss it with Yooni in #corrin on Discord.

**Comment by jobrien97** [comment]
Dspecial changes seem to be missing from the changelog, so I'm not sure what to be looking for

File: fighters/kamui/src/opff.rs [hunk 1]
```diff
@@ -17,8 +17,23 @@ unsafe fn dragon_fang_shot_dash_cancel(boma: &mut BattleObjectModuleAccessor, st
...
+            MotionModule::change_motion(boma, Hash40::new("special_lw_hit"), 0.0, 1.0, false, 0.0, false, false);
+            ArticleModule::change_motion(boma, *FIGHTER_KAMUI_GENERATE_ARTICLE_WATERDRAGON, Hash40::new("special_lw_hit"), true, 0.0);
+        }
+    } else if motion_kind == hash40("special_air_lw") {
+        if frame == 1.0 {
+            MotionModule::change_motion(boma, Hash40::new("special_air_lw_hit"), 0.0, 1.0, false, 0.0, false, false);
+            ArticleModule::change_motion(boma, *FIGHTER_KAMUI_GENERATE_ARTICLE_WATERDRAGON, Hash40::new("special_air_lw_hit"), true, 0.0);
+        }
+    }
+}
```
Comment (jobrien97) [inline fighters/kamui/src/opff.rs:32]: You could just do this in ACMD, no?

**Review by jobrien97 — APPROVED** [review]
lgtm, didnt know downB stuff was WIP

**Pull request merged**